src/target/target-opening-hours/TargetOpeningHours.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { IconInfoCircle, Navigation, Button } from 'hds-react';
+import OpeningPeriodCard from '../opening-period-card/OpeningPeriodCard';
+import Collapse from '../../components/collapse/Collapse';
+import './TargetOpeningHours.scss';
+
+export default function TargetOpeningHours({
+  id,
+}: {
+  id: string;
+}): JSX.Element {
+  interface LanguageOption {
+    label: string;
+    value: string;
+  }
+
+  const languageOptions: LanguageOption[] = [
+    { label: 'Suomeksi', value: 'fi' },
+    { label: 'Svenska', value: 'sv' },
+    { label: 'English', value: 'en' },
+  ];
+  const [language, setLanguage] = useState(languageOptions[0]);
+  const formatSelectedValue = ({ value }: LanguageOption): string =>
+    value.toUpperCase();
+
+  return (
+    <section>
+      <Collapse
+        isOpen
+        collapseContentId={`${id}-opening-hours-section`}
+        title="Toimipisteen aukiolotiedot">
+        <p>
+          Toimipisteen aukiolotietoja muokataan jaksokohtaisesti. Aukiolojaksot
+          voivat olla julkaistuja tai julkaisemattomia. Alla voit selata myös
+          tulevia ja menneitä aukiolojaksoja. Näet alla myös eri kieliversiot
+          valitsemalla kielen valikosta. Huomioithan, että palvelu voi itse
+          valita aukiolojaksojen esitystavan, se ei välttämättä ole alla näkyvän
+          kaltainen.
+        </p>
+        <header className="target-opening-hours-header">
+          <div className="opening-periods-info-container">
+            <h3 className="opening-periods-title">Aukiolojaksot</h3>
+            <IconInfoCircle className="opening-periods-info" />
+          </div>
+          <div className="opening-periods-info-container">
+            <p className="period-count">4 jaksoa</p>
+            <Navigation.LanguageSelector
+              className="opening-periods-language-selector"
+              ariaLabel="Valittu kieli"
+              options={languageOptions}
+              formatSelectedValue={formatSelectedValue}
+              onLanguageChange={setLanguage}
+              value={language}
+            />
+            <Button
```

Comment:
Should we hide all the buttons that do not work? 🤔

Reply:
Nah, I think having the button there is more towards complete app, and then we don't have to introduce new hiding logic to the code. But I agree with the principle; I too would wish we could get to the phase where we write more complete software and pieces, rather than parts that do not work yet. We just lack the required requirements engineering and design for that unfortunately. Also not having the api ready yet does not help. I honestly think I took this task too early to work on; the requirements and design was not ready enough yet.

Reply:
👍